doc/users/github_stats.rst

```diff
@@ -677,7 +677,7 @@ Pull Requests (1003):
 * :ghpull:`7548`: Clarify to_rgba docstring.
 * :ghpull:`7476`: Sticky margins
 * :ghpull:`7552`: Correctly extend a lognormed colorbar
-* :ghpull:`7499`: Improve the the marker table in markers_api documentation
+* :ghpull:`7499`: Improve the marker table in markers_api documentation
```

Comment:
I just fixed this in the PR title so the next time we generate this file it will stay fixed :+1: